inst/AmyloGraph/www/amylograph.css

```diff
@@ -145,22 +133,16 @@ body {
   font-weight: bold;
 }
 
-.ag_protein_details {
-  height: calc(100% - 90px);
-}
-
-.ag_protein_details > .tabbable {
-  max-height: 70%;
-  width: 100%;
-  overflow: scroll;
+#protein_details > * {
+  overflow: auto;
 }
 
-.ag_close_button {
+#btn_close_tab {
```

Comment:
It's not that simple of a replacement, the ID is a bit longer. Perhaps we could get away with not using NS for this button, making the ID predictable? Or shouldn't we?